code/modules/research/xenobiology/crossbreeding/unstable.dm

```diff
@@ -0,0 +1,261 @@
+/obj/item/slimecross/unstable
+	name = "unstable extract"
+	desc = "You can feel rumblin inside..."
+	icon_state = "unstable"
+	effect = "unstable"
+	var/mob/living/carbon/human/human_thrower
+
+/obj/item/slimecross/unstable/throw_at(atom/target, range, speed, mob/thrower, spin, diagonals_first, datum/callback/callback, force, quickstart)
+	if(ishuman(thrower))
+		human_thrower = thrower
+	return ..()
+
+
+/obj/item/slimecross/unstable/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(human_thrower)
+		on_effect(hit_atom)
+	qdel(src)
+
+/obj/item/slimecross/unstable/proc/on_effect(atom/hit_atom)
+	return
+
+/obj/item/slimecross/unstable/grey
+	colour = "grey"
+
+/obj/item/slimecross/unstable/grey/on_effect(atom/hit_atom)
+	var/mob/living/simple_animal/slime/agrro/slimey =  new /mob/living/simple_animal/slime/agrro(get_turf(hit_atom))
+	slimey.Friends[human_thrower] = SLIME_FRIENDSHIP_ATTACK
+	if(ishuman(hit_atom))
+		var/mob/living/carbon/human/humie = hit_atom
+		humie.buckle_mob(slimey)
+
+/obj/item/slimecross/unstable/orange
+	colour = "orange"
+
+/obj/item/slimecross/unstable/orange/on_effect(atom/hit_atom)
+	new /obj/effect/hotspot(hit_atom.drop_location(),200,500)
+
+/obj/item/slimecross/unstable/purple
+	colour = "purple"
+
+/obj/item/slimecross/unstable/purple/on_effect(atom/hit_atom)
+	var/datum/effect_system/smoke_spread/chem/S = new()
+	var/turf/location = get_turf(src)
+
+	// Create the reagents to put into the air
+	create_reagents(20)
+	reagents.add_reagent(/datum/reagent/medicine/regen_jelly,20)
+
+	// Attach the smoke spreader and setup/start it.
+	S.attach(location)
+	S.set_up(reagents, 3, location, silent = TRUE)
+	S.start()
+
+/obj/item/slimecross/unstable/blue
+	colour = "blue"
+
+/obj/item/slimecross/unstable/blue/on_effect(atom/hit_atom)
+	for(var/i in 0 to 5)
+		var/obj/effect/particle_effect/water/W = new /obj/effect/particle_effect/water(get_turf(src))
+		W.Move(get_step(W,pick(GLOB.alldirs)))
+
+	for(var/mob/living/carbon/carbie in range(1,src)){
+		carbie.fire_stacks = -2
+		carbie.ExtinguishMob()
+	}
```

Comment:
```suggestion
	for(var/mob/living/carbon/carbie in range(1,src)){
		carbie.fire_stacks = -2
		carbie.ExtinguishMob()
```

Reply:
god i got used to programming in java lately, that is a big error on my part